backend/common/src/main/java/ai/verta/modeldb/common/reconcilers/Reconciler.java

```diff
@@ -142,4 +144,8 @@ public void insert(T element) {
   public abstract void resync();
 
   protected abstract ReconcileResult reconcile(Set<T> objs);
+
+  public boolean isEmpty() {
+    return !processingIdSet.isEmpty();
+  }
```

Comment:
This seems wrong, no? It's empty if the id set is not empty?

Reply:
Yes, Renamed it with `isNotEmpty`

Reply:
It defeats the purpose of the original comment (https://github.com/VertaAI/modeldb/pull/2708#discussion_r755682768), but ok.